kobo/apps/accounts/tests/test_email_content.py

```diff
@@ -0,0 +1,118 @@
+import random
+
+from django.contrib.auth import get_user_model
+from django.core import mail
+from django.test import TestCase
+from django.urls import reverse
+from rest_framework import status
+
+
+from kobo.apps.accounts.models import EmailContent
+
+
+class EmailContentModelTestCase(TestCase):
+    """
+    These tests are to ensure both the custom activation email and the default
+    activation emails work as expected
+    """
+    def setUp(self) -> None:
+        self.signup_url = reverse('account_signup')
+
+    def test_custom_activation_email_template(self):
+        email_content = EmailContent.objects.create(
+            email_name='email_confirmation_signup_message',
+            section_name='section_one',
+            content='This is some content to test'
+        )
+        email_subject = EmailContent.objects.create(
+            email_name='email_confirmation_signup_message',
+            section_name='subject',
+            content='This is a test subject line'
+        )
+
+        # Using the randomint to make sure that usernames and emails
+        # are unique to ensure the tests will pass when run back to back
+        # as only one email can be requested per username and email every three
+        # minutes
+        username = 'user' + str(random.randint(1, 1000000))
+        email = username + '@example.com'
+        data = {
+            'email': email,
+            'password1': username,
+            'password2': username,
+            'username': username,
+        }
+        request = self.client.post(self.signup_url, data)
+        user = get_user_model().objects.get(email=email)
+        assert request.status_code == status.HTTP_302_FOUND
+        self.client.login(username=user.username, password=user.password)
+        self.client.get(reverse('account_email_verification_sent'))
+        assert len(mail.outbox) == 1
+        assert mail.outbox[0].subject == email_subject.content
+        assert email_content.content in mail.outbox[0].body
+
+    def test_custom_activation_email_template_blank_content(self):
+        email_content = EmailContent.objects.create(
+            email_name='email_confirmation_signup_message',
+            section_name='section_one',
+            content=''
+        )
+        email_subject = EmailContent.objects.create(
+            email_name='email_confirmation_signup_message',
+            section_name='subject',
+            content=''
+        )
+        # Using the randomint to make sure that usernames and emails
```

Comment:
I would fix the issue directly instead of working around it. 

- If you'd like it to not really cache then use `@override_settings(CACHES={'default': {'BACKEND': 'django.core.cache.backends.dummy.DummyCache'}})` 
- If you'd like isolated but functional cache - use LocMemCache. I'm not sure if this will clear between test cases but you could either clear it manually in setUp or set a unique cache location value per test.

That avoids the need for randomness. If you did want to use randomness regardless, that's fine but you should seed it. Importing and using kpi.tests.kpi_test_case.BaseTestCase will automatically set a seed. This creates pseudo random data. It's "random" but will process the same "random" results every time.